api.yml

```diff
@@ -1217,6 +1214,7 @@ components:
             vehicle_type:
```

Comment:
Why are we keeping this? Could we not just re-add it if it becomes necessary again?

Reply:
@hannessolo i just made it optional, so you can leave it out.